falcon/testing/client.py

```diff
@@ -18,7 +18,16 @@
 WSGI callable, without having to stand up a WSGI server.
 """
 
-import json  # Don't use util.json so we can sanity-test compatibility with ujson output
+try:
+    from ujson import dumps as json_dumps
+except ImportError:
+    from json import dumps as json_dumps
+
+try:
```

Comment:
Note that these separate ``try`` ... ``except``s for ``json_dumps`` and ``json_load`` blocks look somewhat ugly, but I was not hitting the second line in coverage checks otherwise.

Maybe they could be listen on comma? Style suggestions/preferences welcome since I am not that familiar with Falcon codebase.